yazi-config/preset/theme.toml

```diff
@@ -182,6 +182,15 @@ rules = [
 	# Documents
 	{ mime = "application/{pdf,doc,rtf,vnd.*}", fg = "green" },
 
+	# Empty files
+	{ mime = "inode/x-empty", fg = "red" },
+
+	# Is-rules
+	{ name = "*" , is = "orphan", fg = "red" },
+	{ name = "*" , is = "link"  , fg = "cyan" },
```

Comment:
"cyan" and "green" are already being used by the images and documents, perhaps we can choose other colors? Or change the images and documents to use different colors?

I think using a strike-through for "orphan" instead of a color might also be a consideration, as this would give us more colors to work with (i.e., red in this case).